les2.js

```diff
@@ -0,0 +1,6 @@
+const treug = (x, y) => {
```

Comment:
bad naming for function name. Your function name must be declarative. I mean that after reading the name of function people must understand what function do in general. And names of function must contain kind of action.
For example: `calcHypotenuse`
Parameters of function also must describe what they store. 
Please, think about naming in all your functions and rename them in according with this ideas